x/cronos/spec/01_concepts.md

```diff
@@ -0,0 +1,49 @@
+ <!-- order: 1 -->
+
+# Concepts
+
+## Cronos
+
+Cronos is an EVM chain in Crypto.org ecosystem. It allows users to deploy and interact with smart contracts. Powered by Ethermint, it is built using cosmos SDK which allows to leverage to full capability of the cosmos ecosystem. It is also connected to the ethereum network with the gravity bridge integration (WIP).
+
+Bridging asset from cosmos or ethereum are automatically converted to a CRC20 asset when moved to Cronos which make it extremely easy to integrate with existing web3 tools.
```

Comment:
maybe a link to CRC20 definition?

Reply:
oh, it's briefly described below -- but may be good to link the actual crc20 doc describing the interface